src/Mod/Draft/draftfunctions/svg.py

```diff
@@ -468,6 +468,8 @@ def get_svg(obj,
                 plane.alignToPointAndAxis_SVG(App.Vector(0, 0, 0),
                                               direction.negative().negative(),
```

Comment:
This could be changed to `App.Vector(direction)`

Reply:
It sure looks like it, but I would like to keep it outside for now as it isn’t related to this PR.
Feel free to make one of your own. I can review it if you tag me :)